state/azure/cosmosdb/cosmosdb_query.go

```diff
@@ -160,9 +160,7 @@ func (q *Query) execute(client *azcosmos.ContainerClient) ([]state.QueryItem, st
 
 	token := ""
 	for queryPager.More() {
-		ctx, cancel := context.WithTimeout(context.Background(), defaultTimeout)
```

Comment:
Please don't remove the background. Still use a context from `ctx` with the timeout

Reply:
Ditto.

Reply:
Fixed.